src/EventStore.Core/Data/StreamMetadata.cs

```diff
@@ -310,5 +311,17 @@ public class StreamMetadata {
 				jsonWriter.WriteEndArray();
 			}
 		}
+
+		public int Size =>
```

Comment:
consider renaming these `Size` getters to something like `ApproximateSize` or `SizeForCache`, since this isn't for marshalling/size required in a buffer etc

Reply:
fixed in: f3d24f346c8b6815142904b38393fb1a9bfb97ba